rollup.config.js

```diff
@@ -77,6 +77,34 @@ export default [
     plugins,
     onwarn,
   },
+  {
+    input: './lib/test/fidelity/components/image-comparison-app.js',
+    output: {
+      file: './dist/image-comparison-app.js',
+      sourcemap: true,
+      format: 'umd',
+      name: 'ImageComparisonApp'
+    },
+    watch: {
+      include: '{lib/test/fidelity/**,lib/third_party/**}',
+    },
+    plugins,
+    onwarn,
+  },
+  {
+    input: './lib/test/fidelity/image-comparison-worker.js',
+    output: {
+      file: './dist/image-comparison-worker.js',
+      sourcemap: true,
+      format: 'umd',
+      name: 'ImageComparisonWorker'
+    },
+    watch: {
+      include: '{lib/test/fidelity/**,lib/third_party/**}',
+    },
+    plugins,
+    onwarn,
+  },
```

Comment:
Build times are already longer than I'd prefer, I don't think we need to rebuild the image comparison on every change during development.

Current build times:

```
time npm run dev

> @google/model-viewer@0.0.7 build /home/jsantell/Dev/model-viewer
> tsc && rollup -c

./lib/model-viewer.js → ./dist/model-viewer.js...
(!) Circular dependency: lib/model-viewer-base.js -> lib/three-components/Renderer.js -> lib/model-viewer-base.js
created ./dist/model-viewer.js in 2.5s

./lib/test/index.js → ./dist/unit-tests.js...
(!) Circular dependency: lib/model-viewer-base.js -> lib/three-components/Renderer.js -> lib/model-viewer-base.js
created ./dist/unit-tests.js in 1.9s

./examples/dependencies/index.js → ./examples/built/dependencies.js...
created ./examples/built/dependencies.js in 694ms

real	0m10.352s
user	0m18.311s
sys	0m0.663s
```

Reply:
Unintentional for it to rebuild on every change. Will fix if I see it doing that locally. But, the watch files are configured as:

```
include: '{lib/test/fidelity/**,lib/third_party/**}'
```

So it shouldn't be doing that 🤷‍♂️ 

As far as build times are concerned, we should always strive to reduce them. But, this item should only build once unless it is being actively dev'd on.

Reply:
Confirmed, it only builds these new targets when I change files that correspond to the configuration.